modin/pandas/test/test_series.py

```diff
@@ -1665,9 +1665,8 @@ def test_lt(data):
 
 @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
 def test_mad(data):
```

Comment:
We should also be testing parameters of the `mad` functionality to ensure correct behavior with those.

Reply:
done